src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -31,16 +33,19 @@
          <wfsServer>http://localhost:8080/geoserver/ows</wfsServer>
          <urlSubstitution key="/mnt/imos-t3/IMOS/opendap/">http://thredds-1-aws-syd.aodn.org.au/thredds/wms/IMOS/</urlSubstitution>
          <urlSubstitution key="^/IMOS/">http://thredds-1-aws-syd.aodn.org.au/thredds/wms/IMOS/</urlSubstitution>
+         <collectionsWithTimeMismatch>^imos:srs.*</collectionsWithTimeMismatch>
```

Comment:
Would it better to call this disregardTimeField or something? to indicate what the effect of setting the flag is?